NCS.DSS.Address/PostAddressHttpTrigger/Function/PostAddressHttpTrigger.cs

```diff
@@ -1,58 +1,56 @@
 using DFC.Common.Standard.Logging;
 using DFC.GeoCoding.Standard.AzureMaps.Model;
 using DFC.HTTP.Standard;
-using DFC.JSON.Standard;
 using DFC.Swagger.Standard.Annotations;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Azure.WebJobs;
-using Microsoft.Azure.WebJobs.Extensions.Http;
 using Microsoft.Extensions.Logging;
 using NCS.DSS.Address.Cosmos.Helper;
 using NCS.DSS.Address.GeoCoding;
 using NCS.DSS.Address.PostAddressHttpTrigger.Service;
 using NCS.DSS.Address.Validation;
-using Newtonsoft.Json;
 using System;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Net;
-using System.Net.Http;
 using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker;
+using NCS.DSS.Address.Helpers;
+using System.Text.Json;
 
 namespace NCS.DSS.Address.PostAddressHttpTrigger.Function
 {
     public class PostAddressHttpTrigger
     {
-        private IResourceHelper _resourceHelper;
-        private IValidate _validate;
-        private IPostAddressHttpTriggerService _addressPostService;
-        private ILoggerHelper _loggerHelper;
-        private IHttpRequestHelper _httpRequestHelper;
-        private IHttpResponseMessageHelper _httpResponseMessageHelper;
-        private IJsonHelper _jsonHelper;
-        private IGeoCodingService _geoCodingService;
+        private readonly IResourceHelper _resourceHelper;
+        private readonly IValidate _validate;
+        private readonly IPostAddressHttpTriggerService _addressPostService;
+        private readonly ILoggerHelper _loggerHelper;
```

Comment:
Remove ILoggerHelper and all its references and use ILogger instead

Reply:
I have removed ILoggerHelper from the Patch trigger class. However, I have kept it in the Post trigger class because it is utilized there, and other team members have also opted to leave it unchanged.